sway-fmt-v2/CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to the Sway Language Formatter
+
+Firstly, thank you for taking interest in advancing the Sway language formatter! This guide will walk you through installation and best practices for contributing to this project.
+
+## Installation
+
+In order to see `swayfmt` in action you will need both `swayfmt` and the [`forc-fmt`](../forc-plugins/forc-fmt/) plugin. As previously stated, you can install these via [`fuelup`](https://github.com/FuelLabs/fuelup) which will install the complete Fuel toolchain, or follow the below instructions to install from source.
+
+**note:** `cargo` is a pre-requisite to this build
+
+1. move to your preferred directory
+2. clone the Sway repo
+3. build from manifest and move the compiled result to your `.cargo/bin` folder
+
+```sh
+git clone https://github.com/FuelLabs/sway.git
+cargo build --manifest-path ~/sway/forc-plugins/forc-fmt/Cargo.toml
+mv ~/sway/target/debug/forc-fmt ~/.cargo/bin
+```
+
+### Testing
+
+Simply run the `forc-fmt` command on the file you want to format:
+
+```sh
+forc fmt
+```
+
+Be sure that your code compiles before executing the command!
+
+### Contribution Guidelines
+
+**Issues:**
+
+- Please check existing issues before opening a new ticket
+- If a bug you find isn't represented by a current ticket, please provide the following details:
+  - `forc-fmt` and `swayfmt` version
+  - `rust` and `cargo` version
+  - operating system and version
```

Comment:
A user may not know how to check the version of any of these tools. Similar to the formatting in a comment above I think this should outline which tool is going to be checked and how to run the corresponding command in order to find the version. Moreover, it should give an example of the output so that the user knows exactly what to copy and paste into the issue because they may copy only some of the information.

It may be helpful to take a look at a visual example of what Sway Apps is doing in terms of its [issue templates](https://github.com/FuelLabs/sway-applications/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml). If you scroll down you'll see it ask for the version, the command to run and it provides an example output

Reply:
Gotcha, I was actually using my memory of that to write this, though I should have just referenced it directly

Reply:
I would like to create a template, though that may end up being tricky since it isn't in its own repo